segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -196,6 +197,19 @@ public PravegaRequestProcessor(StreamSegmentStore segmentStore, TableStore table
 
     //region RequestProcessor Implementation
 
+    @Override
```

Comment:
If this is an Admin Command, the definition of this method should be in `AdminRequestProcessor` and the implementation in `AdminRequestProcessorImpl`. This is why these classes are there, to diverge from `PravegaRequestProcessor` without impacting it.

Reply:
Done!